man/waybar-sway-layout-5.scd

```diff
@@ -0,0 +1,82 @@
+waybar-sway-layout(5)
+
+# NAME
+
+waybar - sway layout module
+
+# DESCRIPTION
+
+The *layout* module displays the current keyboard layout of Sway
+
+# CONFIGURATION
```

Comment:
I am not sure about the different fields here, I picked them from waybar-disk(5). Are they herited from ALabel ?